README.md

```diff
@@ -99,6 +127,7 @@ app.use(bodyParser.urlencoded({extended: true}))
 
 ```text
 Your answer...
+It is a middleware that is used to handle form submissions in Express
 ```
 
```

Comment:
👍 We didn't cover before the checkpoint but `.json()` will parse the data from the incoming request